values/vault-operator/vault-operator-raw.gotmpl

```diff
@@ -124,27 +136,32 @@ resources:
       externalConfig:
         policies:
           - name: allow-read-all-secrets
-            rules: path "secret/*" {
+            rules: |
+              path "secret/*" {
                     capabilities = ["read", "list"]
-                  }
+              }
           - name: allow-list-all-secrets
-            rules: path "secret/*" {
+            rules: |
+              path "secret/*" {
                     capabilities = ["list"]
-                  }
+              }
           - name: allow_sandbox
```

Comment:
I notice we did not enforce style here consistently, could we maybe use hyphens? `allow-sandbox`

Reply:
Right. However, I am not going to change as vault-configurer will not remove old policy and update its reference at vault and then manual intervtion will be needed.